[2024-10-08T21:34:18.963+0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-08T21:34:18.980+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: rf_train.get_data manual__2024-10-08T16:34:14.893269+00:00 [queued]>
[2024-10-08T21:34:18.988+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: rf_train.get_data manual__2024-10-08T16:34:14.893269+00:00 [queued]>
[2024-10-08T21:34:18.988+0500] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-10-08T21:34:18.999+0500] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): get_data> on 2024-10-08 16:34:14.893269+00:00
[2024-10-08T21:34:19.011+0500] {logging_mixin.py:190} WARNING - /home/nicolaspc/anaconda3/envs/mlops_env/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=48110) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-08T21:34:19.012+0500] {standard_task_runner.py:72} INFO - Started process 48168 to run task
[2024-10-08T21:34:19.012+0500] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'rf_train', 'get_data', 'manual__2024-10-08T16:34:14.893269+00:00', '--job-id', '878', '--raw', '--subdir', 'DAGS_FOLDER/train_models.py', '--cfg-path', '/tmp/tmphyeu3wsx']
[2024-10-08T21:34:19.014+0500] {standard_task_runner.py:105} INFO - Job 878: Subtask get_data
[2024-10-08T21:34:19.066+0500] {task_command.py:467} INFO - Running <TaskInstance: rf_train.get_data manual__2024-10-08T16:34:14.893269+00:00 [running]> on host nicolaspc-HP-Pavilion-Laptop-15-eh1xxx
[2024-10-08T21:34:19.145+0500] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Nicolas Pal' AIRFLOW_CTX_DAG_ID='rf_train' AIRFLOW_CTX_TASK_ID='get_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-08T16:34:14.893269+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-08T16:34:14.893269+00:00'
[2024-10-08T21:34:19.146+0500] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-08T21:34:19.342+0500] {base.py:84} INFO - Retrieving connection 'pg_connection'
[2024-10-08T21:34:19.368+0500] {base.py:84} INFO - Retrieving connection 's3_connection'
[2024-10-08T21:34:19.368+0500] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='s3_connection', conn_type='aws') credentials retrieved from login and password.
[2024-10-08T21:34:37.602+0500] {train_models.py:108} INFO - Данные загружены из б/д и сохранены на S3
[2024-10-08T21:34:37.604+0500] {python.py:240} INFO - Done. Returned value was: {'model': 'rf', 'start_timestamp': '20241008 21:34', 'experiment_id': '142', 'get_data_start': '20241008 21:34', 'get_data_end': '20241008 21:34', 'data_shape': (20640, 9)}
[2024-10-08T21:34:37.646+0500] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-08T21:34:37.647+0500] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=rf_train, task_id=get_data, run_id=manual__2024-10-08T16:34:14.893269+00:00, execution_date=20241008T163414, start_date=20241008T163418, end_date=20241008T163437
[2024-10-08T21:34:37.692+0500] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-10-08T21:34:37.708+0500] {local_task_job_runner.py:240} INFO - ::endgroup::
